### ISSUE TITLE
How to send unsolicited messages to the Chat Bot

### ISSUE THREAD

**Issue by luigisaetta** [guidone/node-red-contrib-chatbot#34]
Hi,
As platform, I'm using Telegram.

As part of a HomeAutomation concept system, I was thinking to add also a ChatBot as interface. With NodeRED and your module it has been easy to create a prototype.
What I'm not been able to do is to send a message to the Bot without a TelegramReceiver.
The problem is that i need to set up the context. Simple with "transport", but then I get an error of the kind: "msg.char() is not a function". It is something in the context set by the Receiver that I need to add.

In short, to be able to use if for notification (for example: an event on the Home site happen, I want to notify an alert using Telegram). Would be nice to document how to manage this UseCase.

**Comment by guidone** [comment]
@luigisaetta perhaps the node you need is the "conversation" node, yes it's missing in the documentation

**Comment by luigisaetta** [comment]
@guidone : thanks for the answer.
There is still something I'm missing
I start the flow with an inject node (to simulate an event), then I have a conversation node with a chatId (taken from debug of another working flow), a message where I set the Message and then a Telegram Sender. Don't work, I'm still getting msg.chat() is not a function. What am I missing?

Anyway, compliments for the module.... really useful.

**Comment by guidone** [comment]
@luigisaetta Thanks, perhaps there's a regression, let me check

**Comment by guidone** [comment]
@luigisaetta ok, install version 0.6.8, it should be fixed

**Comment by luigisaetta** [comment]
@guidone: yes it works. Thanks !
